src/Features/Core/Portable/AddConstructorParametersFromMembers/State.cs

```diff
@@ -38,29 +30,49 @@ private class State
 
             private bool TryInitialize(
                 AddConstructorParametersFromMembersCodeRefactoringProvider service,
-                Document document,
-                TextSpan textSpan,
-                ImmutableArray<ISymbol> selectedMembers,
-                CancellationToken cancellationToken)
+                ImmutableArray<ISymbol> selectedMembers)
             {
                 if (!selectedMembers.All(IsWritableInstanceFieldOrProperty))
                 {
                     return false;
                 }
 
-                this.TextSpan = textSpan;
-                this.SelectedMembers = selectedMembers;
-                this.ContainingType = this.SelectedMembers[0].ContainingType;
-                if (this.ContainingType == null || this.ContainingType.TypeKind == TypeKind.Interface)
+                ContainingType = selectedMembers[0].ContainingType;
+                if (ContainingType == null || ContainingType.TypeKind == TypeKind.Interface)
```

Comment:
i would prefer you keep the 'this.'.  But i'm not going to push hard if you prefer this.  'this.' just makes it super simple to know that something is stateful.  it would also make this PR have less diffs.